docs/user/getting_started.rst

```diff
@@ -14,6 +14,14 @@ and visualize the changes being made in GeoServer.
 
 See :doc:`/tutorials/geomesa-quickstart-accumulo`.
 
+
+GeoDocker: Bootstrapping GeoMesa Accumulo and Spark on AWS
+----------------------------------------------------------
+
+Getting started with spatio-temporal analysis with GeoMesa, Accumulo, and Spark on Amazon Web Services (AWS) is incredibly simple, thanks to the `Geodocker <https://github.com/geodocker/geodocker-accumulo-geomesa>`_ project.  The guide below describes how to bootstrap a GeoMesa Accumulo cluster using Amazon ElasticMapReduce (EMR) and Docker and ingesting and querying some sample GDELT data.
```

Comment:
[picky] 

> The guide below describes how to bootstrap a GeoMesa Accumulo cluster using Amazon ElasticMapReduce (EMR) and Docker and ingesting and querying some sample GDELT data.

Seems clunky with 3 'and's even though it's technically correct, it could be rephrased as:

> The guide below describes how to bootstrap a GeoMesa Accumulo cluster using Amazon ElasticMapReduce (EMR) and Docker in order to ingest and query some sample GDELT data.

This sentence appears twice, here and in geodocker-geomesa-spark-on-aws.rst.